.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/MergedDateTimeParserChs.cs

```diff
@@ -115,10 +115,10 @@ public new ParseResult Parse(ExtractResult er, DateObject refTime)
                 pr.Value = val;
             }
 
-            pr.Value = DateTimeResolution(pr, hasBefore, hasAfter, hasSince);
+            pr.Value = DateTimeResolution(pr);
 
             //change the type at last for the after or before mode
-            pr.Type = $"{ParserTypeName}.{DetermineDateTimeType(er.Type, hasBefore, hasAfter, hasSince)}";
+            pr.Type = $"{ParserTypeName}.{DetermineDateTimeType(er.Type, hasMod: hasBefore || hasAfter || hasSince)}";
```

Comment:
Please declare a var for hasModifier. It makes code easier to read.

Reply:
Fixed.